pymongo/operations.py

```diff
@@ -547,7 +547,7 @@ def document(self) -> dict[str, Any]:
 class SearchIndexModel:
     """Represents a search index to create."""
 
-    __slots__ = "__document"
+    __slots__ = ("__document",)
```

Comment:
Found a bug!

Reply:
Not a bug :). `__slots__` can be assigned a single attr name or a tuple of names: https://docs.python.org/3/reference/datamodel.html#object.__slots__

Reply:
Ah, shucks.